lib/pbench/agent/tool_meister.py

```diff
@@ -1770,7 +1791,9 @@ def sysinfo(self, data: Dict[str, str]) -> int:
         return failures
 
 
-def get_logger(PROG: str, daemon: bool = False, level: str = "info") -> logging.Logger:
+def get_logger(
+    PROG: str, is_daemon: bool = False, level: str = "info"
+) -> logging.Logger:
```

Comment:
Why did this get wrapped now and not before it landed in the tree?

Reply:
Apparently the expansion from `daemon` to `is_daemon` was enough to bother `black`!